plutus-core/plutus-core/test/CostModelInterface/defaultCostModelParams.json

```diff
@@ -13,15 +13,60 @@
   "appendString-memory-arguments-slope": 1,
   "bData-cpu-arguments": 1000,
   "bData-memory-arguments": 32,
+  "blake2b_224-cpu-arguments-intercept": 117366,
+  "blake2b_224-cpu-arguments-slope": 10475,
+  "blake2b_224-memory-arguments": 4,
   "blake2b_256-cpu-arguments-intercept": 117366,
   "blake2b_256-cpu-arguments-slope": 10475,
   "blake2b_256-memory-arguments": 4,
+  "bls12_381_G1_add-cpu-arguments": 1046420,
```

Comment:
hmmmm, weird that these were missing before??

Reply:
Yes, I wondered about this.

Reply:
Actually I think that's OK.  This is used by the tests for [the code that updates the cost model ](https://github.com/input-output-hk/plutus/blob/f8d5adf0e9b558a63a233cb4126eee58a5eebb0c/plutus-core/plutus-core/test/CostModelInterface/Spec.hs#L30) using the flattened parameters.  The tests are self-contained and it doesn't matter exactly _what_ cost model is used as long as it's sensible.  It won't do any harm to extend it though.